screens/ConditionsForm.js

```diff
@@ -179,14 +179,15 @@ const ConditionsForm = ({ navigation })  => {
     const routeParams = {
       lat: geoCoordinates.lat,
       long: geoCoordinates.lon,
-      species: species.name,
+      species: species.name.toLowerCase(),
```

Comment:
We shouldn't do this because we need the names to be centralized, instead of doing this, can you change the fish ids to be the name of the fish in camel case (ex: "Brown Trout would be id: brownTrout") instead of number, this is the way they are written in the json files.